src/aal/aal.h

```diff
@@ -59,6 +68,13 @@ namespace snmalloc
      */
     static inline uint64_t tick()
     {
+      if constexpr((Arch::aal_features & NoCpuCycleCounters) == NoCpuCycleCounters)
+      {
+        struct timespec n = {0, 0ul};
+        clock_gettime(CLOCK_MONOTONIC, &n);
```

Comment:
This is posix specific, can you add an 
```
#ifdef _WIN32
#  error "Implement Windows Version"
#else
   ....
#endif`
```
And when someone wants this for Windows on Arm they should get a good error message.  This should also fix the build issues for Windows Clang.

Reply:
Could we use 
```
std::chrono::high_resolution_clock::now()
```
this is actually cross platform.  Then it should work on any platform.